Tests/ClientTests/Coordinators/SettingsCoordinatorTests.swift

```diff
@@ -234,6 +234,24 @@ final class SettingsCoordinatorTests: XCTestCase {
         XCTAssertTrue(mockRouter.pushedViewController is CreditCardSettingsViewController)
     }
 
+    func testPasswordSettings_showsPasswordListVC() {
```

Comment:
I think having test for `handle(route:` for the password enum case would be great too